release/models/gnsi/openconfig-gnsi-acctz.yang

```diff
@@ -207,29 +209,26 @@ module openconfig-gnsi-acctz {
 
   grouping grpc-server-acctz-counters {
     description
-      "A collection of counters from the gNSI.acctz module.";
+      "A collection of counters from the gNSI.acctz module. This collection
+      augments the existing certz grpc counters";
 
-    container acctz-counters {
+    container acctz {
       config false;
       description
-        "A collection of counters from the gNSI.acctz module
-        for acctz clients and sources.";
-
-      container state {
-        description
-          "Operational state relating to acctz-counters.";
-
-        leaf counters-last-cleared {
-          type oc-types:timeticks64;
-          description
-            "The last time that the counters were cleared (reset to
-            zero). This value is reported as nanoseconds since epoch
-            (January 1st, 1970 00:00:00 GMT).";
-        }
+        "A collection of counters from the gNSI.acctz module. This collection
+        augments the existing certz grpc counters";
```

Comment:
```suggestion
        augments the existing grpc counters when GNSI is enabled.";
```

Reply:
Thanks, I appreciate the suggestions. I went something a little different to emphasize this grouping applies to the list of grpc-servers.